pom.xml

```diff
@@ -1571,6 +1571,31 @@ flexible messaging model and an intuitive client API.</description>
         </executions>
       </plugin>
 
+      <plugin>
+        <groupId>pl.project13.maven</groupId>
+        <artifactId>git-commit-id-plugin</artifactId>
+        <version>${git-commit-id-plugin.version}</version>
+        <executions>
+          <execution>
+            <id>git-info</id>
+            <goals>
+              <goal>revision</goal>
+            </goals>
+          </execution>
+        </executions>
+        <configuration>
+          <skip>true</skip>
```

Comment:
Just thinking whether `pulsar-common`'s usage of `git-commit-id-plugin` continues to work as expected. It should since by default all configuration is replaced if it's defined in a child pom.xml?